fabulaws-config.yml

```diff
@@ -178,14 +178,14 @@
   volume_sizes:
     production:
       cache: 10
-      db-master: 100
-      db-slave: 100
+      db-master: 250
+      db-slave: 250
       web: 10
       worker: 50
     testing:
       cache: 10
-      db-master: 100
-      db-slave: 100
+      db-master: 250
+      db-slave: 250
       web: 10
```

Comment:
+1

Sent from my mobile.
On Feb 4, 2016 9:11 PM, "Vinod Kurup" <notifications@github.com> wrote:

> In fabulaws-config.yml
> <https://github.com/caktus/django-opendebates/pull/53#discussion_r51968875>
> :
>
> >        web: 10
> >        worker: 50
> >      testing:
> >        cache: 10
> > -      db-master: 100
> > -      db-slave: 100
> > +      db-master: 250
> > +      db-slave: 250
> >        web: 10
>
> while we're here, should we bump up the web size a bit, since they're
> currently 70% full?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/caktus/django-opendebates/pull/53/files#r51968875>.
>